examples/coordinates/plot_galactocentric-frame.py

```diff
@@ -77,16 +77,17 @@
 # changes values using the keywords ``galcen_distance``, ``galcen_v_sun``, and
 # ``z_sun`` which set the sun-Galactic center distance, the 3D velocity vector
 # of the sun, and the height of the sun above the Galactic midplane,
-# respectively. The velocity of the sun must be specified as a
-# `~astropy.coordinates.CartesianDifferential` instance, as in the example
-# below. Note that, as with the positions, the Galactocentric frame is a
-# right-handed system - the x-axis is positive towards the Galactic center, so
-# ``v_x`` is opposite of the Galactocentric radial velocity:
-
-v_sun = coord.CartesianDifferential([11.1, 244, 7.25]*u.km/u.s)
-gc_frame = coord.Galactocentric(galcen_distance=8*u.kpc,
-                                galcen_v_sun=v_sun,
-                                z_sun=0*u.pc)
+# respectively. The velocity of the sun can be specified as an
+# `~astropy.units.Quantity` object with velocity units and is interepreted as a
+# Cartesian velocity, as in the example below. Note that, as with the positions,
+# the Galactocentric frame is a right-handed system (i.e. the Sun is at negative
+# x values) so ``v_x`` is opposite of the Galactocentric radial velocity:
+
+v_sun = [11.1, 244, 7.25] * u.km/u.s  # [vx, vy, vz]
```

Comment:
It is good practice to create the composite unit first, right? This would also apply to other similar usage in this file.
```suggestion
v_sun = [11.1, 244, 7.25] * (u.km / u.s)  # [vx, vy, vz]
```